packages/core/src/routes/sign-in-experience.ts

```diff
@@ -0,0 +1,44 @@
+import { SignInExperiences } from '@logto/schemas';
+import { object, string } from 'zod';
+
+import koaGuard from '@/middleware/koa-guard';
+import {
+  findDefaultSignInExperience,
+  updateSignInExperienceById,
+} from '@/queries/sign-in-experience';
+
+import { AuthedRouter } from './types';
+
+export default function SignInExperiencesRoutes<T extends AuthedRouter>(router: T) {
```

Comment:
same here, should we stick to camel-case for function name `SignInExperiencesRoutes`?